Sources/UIComponents/Resources/Font/Font.swift

```diff
@@ -0,0 +1,98 @@
+//
+//  File.swift
+//  
+//
+//  Created by Nicolai Dam on 09/12/2021.
+//
+
+import SwiftUI
+import UIKit
+
+
+public class StyleBundle {}
+
+class CurrentBundleFinder {}
+
+public enum AcademyName: String, CaseIterable {
+    case skat_black = "AcademySans-Black"
+    case skat_regular = "AcademySans-Regular"
+    case skat_bold = "AcademySans-Bold"
+    case skat_demiBold = "AcademySans-Demibold"
+    case skat_medium = "AcademySans-Medium"
```

Comment:
Why it's called with _? can you make it camel case?

Reply:
I just stealed it from another project - good point, will make it camelcase